app/assets/javascripts/hera/modules/search.js

```diff
@@ -0,0 +1,51 @@
+document.addEventListener('turbolinks:load', function () {
+  $('[data-behavior~=form-search]').hover(function () {
+    $(this).find('[data-behavior~=search-query]').focus();
+  });
+
+  $('[data-behavior~=search-button]').on('click', function (e) {
+    e.preventDefault();
+    submitSearch($(this).parents('form'));
+  });
+
+  $('[data-behavior~=search-query]').on('keypress', function (e) {
+    if (e.which === 13) {
```

Comment:
can we make the magic number a variable like we did with `mobileBreakpoint` in sidebar.js